packages/edit-site/src/components/page-actions/index.js

```diff
@@ -9,20 +9,24 @@ import { moreVertical } from '@wordpress/icons';
  * Internal dependencies
  */
 import TrashPageMenuItem from './trash-page-menu-item';
+import RenameMenuItem from '../rename-menu-item';
 
-export default function PageActions( { postId, toggleProps, onRemove } ) {
+export default function PageActions( { onRemove, page, toggleProps } ) {
 	return (
 		<DropdownMenu
 			icon={ moreVertical }
 			label={ __( 'Actions' ) }
 			toggleProps={ toggleProps }
 		>
-			{ () => (
+			{ ( { onClose } ) => (
 				<MenuGroup>
-					<TrashPageMenuItem
-						postId={ postId }
-						onRemove={ onRemove }
-					/>
+					<RenameMenuItem item={ page } onClose={ onClose } />
```

Comment:
`item` doesn't specifically describe the value being passed. I'd go with `post` which is the technical name for the entity record. I'd also rename `RenameMenuItem` to `RenamePostMenuItem` if it's not too late to do that.

```js
<RenamePostMenuItem post={ post } onClose={ onClose } />
```